oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java

```diff
@@ -88,6 +88,8 @@ public class StorageModuleElasticsearchConfig extends ModuleConfig {
      * @since 8.7.0 This setting affects all traces/logs/metrics/metadata flush policy.
      */
     private int bulkActions = 5000;
+
+    private int batchOfBytes = 1024 * 1024 * 5;
```

Comment:
```suggestion
    /**
     * @since 9.4.0 A threshold to control the max body size of ElasticSearch Bulk flush. 
     */
    private int batchOfBytes = 1024 * 1024 * 5;
```

Reply:
Please update the `application.yml` comments accordingly.

Reply:
> Please update the `application.yml` comments accordingly.

done